openedx/features/partners/views.py

```diff
@@ -21,9 +27,9 @@ def dashboard(request, slug):
         raise Http404('Your partner is not properly registered')
 
 
+@csrf_exempt
 @require_http_methods(["POST"])
 @sensitive_post_parameters('password')
-@csrf_exempt
```

Comment:
Lets revert this unnecessary position change of `@csrf_exempt`. It will create conflict because in another [PR-1273](https://github.com/philanthropy-u/edx-platform/pull/1273) i have removed this annotation

Reply:
Fixed.